components/feature/session-bundling/src/main/java/mozilla/components/feature/session/bundling/SessionBundleStorage.kt

```diff
@@ -130,12 +130,16 @@ class SessionBundleStorage(
 
     /**
      * Returns the last saved [SessionBundle] instances (up to [limit]) as a [LiveData] list.
+     *
+     * @param since (Optional) Unix timestamp (milliseconds). If set this method will only return [SessionBundle]
+     * instances that have been saved since the given timestamp.
+     * @param limit (Optional) Maximum number of [SessionBundle] instances that should be returned.
      */
-    fun bundles(limit: Int = 20): LiveData<List<SessionBundle>> {
+    fun bundles(since: Long = 0, limit: Int = 20): LiveData<List<SessionBundle>> {
```

Comment:
fun fact:  If [I convert 0 to a date](http://timestamp.online/) is `12/31/1969, 7:00:00 PM` luckily all our bundles are going to be newer than that 😆

Reply:
Yeah, timezones are hard :D